cscs-checks/compile/libsci_resolve.py

```diff
@@ -26,19 +25,15 @@ def __init__(self, module_name):
 
         self.module_name = module_name
         self.module_version = {
-            'craype-accel-nvidia20': 'nv20',
             'craype-accel-nvidia35': 'nv35',
             'craype-accel-nvidia60': 'nv60'
         }
         self.compiler_version = {
-            'dom':   '49',
+            'dom':   '71',
             'daint': '49',
```

Comment:
I would change Daint as well to have the test ready for whenever the upgrade happens.